internal/run/run.go

```diff
@@ -887,12 +887,6 @@ func resolveDocument(ctx context.Context, d workflow.Document, outputLocation *s
 }
 
 func resolveRemoteDocument(ctx context.Context, d workflow.Document, outPath string) (string, error) {
-	log.From(ctx).Infof("Downloading %s... to %s\n", d.Location, outPath)
```

Comment:
We can't optimistically log because we don't know the ultimate filename. This log call is moved to the end of this function.